spk/fossil-scm/Makefile

```diff
@@ -0,0 +1,22 @@
+SPK_NAME = fossil-scm
+SPK_VERS = 1.0
+SPK_REV = 1
+SPK_ICON = src/fossil-scm.png
+
+DEPENDS = cross/fossil-scm
+
+MAINTAINER = SynoCommunity
+DISPLAY_NAME = Fossil SCM
+
+OPTIONAL_DESC =
+
+DESCRIPTION = "Fossil is a simple, high-reliability, distributed software configuration management system."
+STARTABLE = no
+CHANGELOG = "Initial version."
+
+HOMEPAGE = https://github.com/SynoCommunity/spksrc/wiki/FAQ-FossilSCM
```

Comment:
the synocommunity wiki is used as homepage for synocli-* packages to collect the information of the individual cli tools.
Usualy this is the homepage/documentation of the packaged software.
use: https://fossil-scm.org/home/doc/trunk/www/index.wiki